Applications/APRS/aprs.py

```diff
@@ -12,17 +12,10 @@
 import logging.config
 import threading
 import ConfigParser
-import os
-import sys
 import socket
 import requests
 from time import sleep
 
-# Can we clean this up?
```

Comment:
Haha wow this is a good find. Totally based APRS off of Telemetry and forgot to remove the FaradayIO import code as it isn't used. @kb1lqd